src/components/DownloadReleases/DownloadTable.tsx

```diff
@@ -39,21 +40,21 @@ const DownloadTable = ({ releases }: Props): JSX.Element => (
 
           return (
             <tr key={release}>
-              <td>
+              <th>
```

Comment:
Changing the row from td to th, isn't that a semantic issue?

Reply:
Good catch, shouldn't be for `tbody` but for `thead`I think that's correct. Let me double-check.